src/window/config.c

```diff
@@ -118,6 +170,24 @@ static void toggle_switch(int key, int param2)
     window_invalidate();
 }
 
+static void set_language(int index)
+{
+    if (index == 0) {
+        data.string_values[CONFIG_STRING_UI_LANGUAGE_DIR][0] = 0;
+    } else {
+        strncpy(data.string_values[CONFIG_STRING_UI_LANGUAGE_DIR], data.language_options_utf8[index], CONFIG_STRING_VALUE_MAX);
+    }
+    data.selected_language_option = index;
+}
+static void button_language_select(int param1, int param2)
```

Comment:
Nitpick: this is not a nitpick. Consequent formatting is important :)

Reply:
Actually, this is intended, since the callback function at the top is intended to be used from the function at the bottom only. In more functional languages this callback function would be a lambda function within `button_language_select`.

Reply:
I have to be honest, while I understand your reasoning I still think it looks better with the empty line between the functions.

Reply:
Fair enough, I'll remove it.